### PR TITLE
Btat 9224 Play 2.8 upgrade

### PR DESCRIPTION
**Pull request by JoNB7777**
# Pull Request Checklist
* [x] Branch is rebased with master
* [ ] Added Unit Tests for changed functionality
* [x] Ran `sbt clean compile coverage test it:test coverageReport`, all tests pass and coverage meets minimum
* [ ] Checked libraries are up to date with `sbt validate`. (If applicable) library updates are done in a separate, single commit
* [x] Ran [change-vat-acceptance-tests](https://github.com/hmrc/change-vat-acceptance-tests) (see README of repo)
* [ ] Ran [change-vat-performance-tests](https://github.com/hmrc/change-vat-performance-tests) (see README of repo)
* [ ] (If applicable) raised app-config-* PRs
* [ ] (If applicable) ran Wave test for changes to views
* [ ] (If applicable) added Integration Tests

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -61,26 +60,25 @@ lazy val coverageSettings: Seq[Setting[_]] = {
 
 val compile: Seq[ModuleID] = Seq(
   ws,
-  "uk.gov.hmrc"   %% "bootstrap-frontend-play-26"      % "5.0.0",
+  "uk.gov.hmrc"   %% "bootstrap-frontend-play-28"      % "5.3.0",
   "uk.gov.hmrc"   %% "govuk-template"                  % "5.66.0-play-26",
```
Comment (william-hatzar) [inline build.sbt:64]: This can be removed as it is no longer needed

File: build.sbt [hunk 2]
```diff
@@ -61,26 +60,25 @@ lazy val coverageSettings: Seq[Setting[_]] = {
...
-  "uk.gov.hmrc"   %% "play-ui"                         % "9.2.0-play-26",
```
Comment (william-hatzar) [inline build.sbt:66]: Can be bumped to play 28

File: build.sbt [hunk 3]
```diff
@@ -61,26 +60,25 @@ lazy val coverageSettings: Seq[Setting[_]] = {
...
+  "uk.gov.hmrc"   %% "play-ui"                         % "9.4.0-play-26",
   "uk.gov.hmrc"   %% "play-partials"                   % "7.1.0-play-26",
```
Comment (william-hatzar) [inline build.sbt:66]: can be bumped to play 28

File: build.sbt [hunk 4]
```diff
@@ -61,26 +60,25 @@ lazy val coverageSettings: Seq[Setting[_]] = {
...
-  "uk.gov.hmrc"   %% "play-language"                   % "4.12.0-play-26",
```
Comment (william-hatzar) [inline build.sbt:68]: Same as the others

File: build.sbt [hunk 5]
```diff
@@ -61,26 +60,25 @@ lazy val coverageSettings: Seq[Setting[_]] = {
...
-  "uk.gov.hmrc"   %% "play-frontend-govuk"             % "0.71.0-play-26",
```
Comment (william-hatzar) [inline build.sbt:69]: Same as the others

File: build.sbt [hunk 6]
```diff
@@ -61,26 +60,25 @@ lazy val coverageSettings: Seq[Setting[_]] = {
...
-  "uk.gov.hmrc"   %% "play-frontend-hmrc"              % "0.57.0-play-26",
```
Comment (william-hatzar) [inline build.sbt:70]: Same as the others

File: app/utils/LoggerUtil.scala [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+import play.api.Logging
+
+// $COVERAGE-OFF$
+
+object LoggerUtil extends Logging {
+
+  def logInfo(content: String): Unit = logger.info(content)
+  def logDebug(content: String): Unit = logger.debug(content)
+  def logWarn(content: String): Unit = logger.warn(content)
+
```
Comment (william-hatzar) [inline app/utils/LoggerUtil.scala:28]: These logs don't seem to be showing in the terminal when I run through the journey

File: build.sbt (hunk 1 above)
Comment (william-hatzar) [inline build.sbt:64]: This can be removed

File: build.sbt [hunk 8]
```diff
@@ -61,26 +60,25 @@ lazy val coverageSettings: Seq[Setting[_]] = {
 
 val compile: Seq[ModuleID] = Seq(
   ws,
-  "uk.gov.hmrc"   %% "bootstrap-frontend-play-26"      % "5.0.0",
+  "uk.gov.hmrc"   %% "bootstrap-frontend-play-28"      % "5.3.0",
   "uk.gov.hmrc"   %% "govuk-template"                  % "5.66.0-play-26",
-  "uk.gov.hmrc"   %% "play-ui"                         % "9.2.0-play-26",
+  "uk.gov.hmrc"   %% "play-ui"                         % "9.4.0-play-26",
```
Comment (william-hatzar) [inline build.sbt:65]: Can be bumped to play-28

File: build.sbt (hunk 3 above)
Comment (william-hatzar) [inline build.sbt:66]: Can be bumped to play-28

File: build.sbt [hunk 9]
```diff
@@ -61,26 +60,25 @@ lazy val coverageSettings: Seq[Setting[_]] = {
...
   "uk.gov.hmrc"   %% "play-partials"                   % "7.1.0-play-26",
-  "uk.gov.hmrc"   %% "play-language"                   % "4.12.0-play-26",
-  "uk.gov.hmrc"   %% "play-frontend-govuk"             % "0.71.0-play-26",
-  "uk.gov.hmrc"   %% "play-frontend-hmrc"              % "0.57.0-play-26",
+  "uk.gov.hmrc"   %% "play-language"                   % "5.0.0-play-26",
```
Comment (william-hatzar) [inline build.sbt:67]: Same as the others

File: build.sbt [hunk 10]
```diff
@@ -61,26 +60,25 @@ lazy val coverageSettings: Seq[Setting[_]] = {
...
+  "uk.gov.hmrc"   %% "play-frontend-govuk"             % "0.73.0-play-26",
```
Comment (william-hatzar) [inline build.sbt:68]: Same as the others

File: build.sbt [hunk 11]
```diff
@@ -61,26 +60,25 @@ lazy val coverageSettings: Seq[Setting[_]] = {
...
+  "uk.gov.hmrc"   %% "play-frontend-hmrc"              % "0.67.0-play-26",
```
Comment (william-hatzar) [inline build.sbt:69]: Same as the others

**Review by SheilaTran — CHANGES_REQUESTED** [review]
Will has done such a good job reviewing! My only comment is to upgrade scalatest to 3.2.9 as it still compiles fine

**Review by SheilaTran — APPROVED** [review]

**Pull request merged**